qemu/tests/virtio_fs_share_data.py

```diff
@@ -455,7 +455,24 @@ def viofs_svc_stop_start(action, cmd, expect_status):
                     session.cmd(cmd_make % fs_dest, io_timeout)
                     status, output = session.cmd_status_output(
                         cmd_pjdfstest % fs_dest, io_timeout)
-                    if status != 0:
+                    failed_test = output.split("-------------------")[1]\
+                                        .split("Files=")[0]
+                    # ignore the specific failed cases from pjdfstest
+                    ignore_cases = ["pjdfstest/tests/rename",
```

Comment:
I think we'd better make it more accurate.
pjdfstest/tests/rename/21.t
pjdfstest/tests/symlink/03.t